schema/borrow-schema.py

```diff
@@ -128,9 +129,8 @@ def compare(actual, infra_list, ocds_list, prefix, suffix):
         sys.exit('{prefix} is missing {items}: remove from infra_{suffix} in borrow-schema.py?'.format(
             items=', '.join(removed), prefix=prefix, suffix=suffix))
 
-
```

Comment:
Please run `flake8 --max-line-length 119`. Not sure why this didn't error in the test. This newline is needed to conform to PEP8.

Reply:
Fixed in https://github.com/open-contracting/infrastructure/pull/222/commits/05c78ec268a229022bb35c0d373e2e8da7db9fdf